Packs/GoogleChronicleBackstory/IndicatorFields/incidentfield-chronicleassetsummary.json

```diff
@@ -0,0 +1,42 @@
+{
+ "id": "indicator_chronicleassetsummary",
+ "version": -1,
+ "modified": "2020-09-22T17:04:57.566588552+05:30",
+ "name": "ChronicleAsset Summary",
+ "ownerOnly": false,
+ "description": "Summary of the event types of the events fetched for all the possible asset identifiers.",
+ "cliName": "chronicleassetsummary",
+ "type": "grid",
+ "closeForm": false,
+ "editForm": false,
+ "required": false,
+ "neverSetAsRequired": false,
+ "isReadOnly": false,
+ "useAsKpi": false,
+ "locked": false,
+ "system": false,
+ "content": true,
+ "group": 2,
+ "hidden": false,
+ "associatedToAll": true,
```

Comment:
Should be associate to Chronicle type and not to all.

Reply:
@altmannyarden You are right. We have made this change.